File: python/fizzbuzz.py
class FizzBuzz:
    def take(self, number):
        if number % 3 == 0 and number % 5 == 0:
            return 'fizzbuzz'
        elif number % 3 == 0:
            return 'fizz'
        elif number % 5 == 0:
            return 'buzz'

        return number

File: php/fizzbuzz.php
<?php

class FizzBuzz {
    function take( $number ) {
        if ( 0 == $number % 3 and 0 == $number % 5 ) {
            return "FizzBuzz";
        } elseif ( 0 == $number % 3 ) {
            return "Fizz";
        } elseif ( 0 == $number % 5 ) {
            return "Buzz";
        } else {
            return $number;
        }
    }
}
